3rdparty/haskell/BUILD.ghc-lib-parser

```diff
@@ -6,7 +6,7 @@ haskell_library(
   name = "lib",
   visibility = ["//visibility:public"],
   srcs = glob(
-    ["**/*.hs", "**/*.hsc", "**/*.hs-boot"],
+    ["**/*.hs", "**/*.hs-boot"], # No .hsc files here!
```

Comment:
I don’t think that is a good idea. If we don’t want to use the `.hsc` files we shouldn’t be including them in the sdist tarball. This is just asking to break once we try to switch back to getting this from Hazel or hopefully soon via the Stack snapshot mechanism in `rules_haskell`.